cfgov/tccp/jinja2/tccp/includes/card_list.html

```diff
@@ -51,9 +51,23 @@
 <div class="o-card-group">
     <div class="o-card-group__cards">
         {%- for card in cards %}
+        {%- set card_purchase_apr = apr_range(
+            card.purchase_apr_for_tier_min,
+            card.purchase_apr_for_tier_max,
+            asterisk=true
```

Comment:
How is the asterisk read by a screen reader when this is used in the `aria-label`? If it sounds weird, I was going to suggest using an `asterisk=false` version in the `aria-label`. But if it's also read awkwardly in the card content, maybe we can update the `apr_range` macro to make the asterisk `aria-hidden` everywhere so it's never read.

Reply:
Great point, the asterisk does not read well either on the results page or the card detail pages. I'm thinking we should either migrate to a [footnote format](https://www.jamesrmeyer.com/otherstuff/easy-footnotes-for-web-pages) (e.g. `[1]`) that links to the explanatory text *or* we can try replacing the asterisks with the full explanatory text for screenreaders only. I'll experiment and open another PR.